cmd/troubleshoot/builder.go

```diff
@@ -212,8 +211,7 @@ func getAllDynakubesInNamespace(ctx context.Context, log logd.Logger, apiReader
 	}
 
 	if len(dynakubes.Items) == 0 {
-		err = fmt.Errorf("no Dynakubes found in namespace '%s'", namespaceName)
-		logErrorf(log, err.Error())
+		logErrorf(log, "no Dynakubes found in namespace '%s'", namespaceName)
 
 		return nil, err
```

Comment:
Make it more obvious what's actually returned here or set `err` to some real error:
```suggestion
		return nil, nil
```

Reply:
it's fair. let me fix it.

Reply:
FYI:
`getAllDynakubesInNamespace` is not actually used in live code
it's only used in `getDynakubes` but that function is only used in tests

the code that actually runs (and does basically the same):
https://github.com/Dynatrace/dynatrace-operator/blob/34f5bb84e96bd79483d37d5f695317928e46a3e6/cmd/troubleshoot/builder.go#L109-L114

Reply:
Ticket to fix it seperatly. https://dt-rnd.atlassian.net/browse/K8S-10962

Reply:
Ticket to fix it seleratly https://dt-rnd.atlassian.net/browse/K8S-10962